infrastructure/ami/scripts/install-huggingface-libraries.sh

```diff
@@ -20,6 +20,8 @@ pip install --upgrade --no-cache-dir \
     "jinja2==3.1.2" \
     "attrs==23.1.0"
 
+# Temporary fix for the issue: https://github.com/huggingface/optimum-neuron/issues/142
+pip install -U optimum
```

Comment:
Looking tat the logs, it installs 1.18, and then when installing optimum-neuron, it seems to use 1.17. I am completely lost with this installation process.
Anyway, it seems to work but we need to clean this up.

Reply:
I think something that we should do is that we always keep optimum-neuron setup compatible with the latest optimum, in this way we are more comfortable to be able to install optimum neuron via either `optimum[neuronx]` or `optimum[neuron]` extra...